docs/create-scenario.md

```diff
@@ -41,4 +38,24 @@ _Core Components_
 
-## Testing
+### Scenario Folder structure
+
+![Scenario Structure](./scenario-structure.png)
+
+Scenarios live in the `simulator/simulation-scripts/scenario/` directory. Minimally they must contain:
+
+- _scenario name_: Substitute "scenario-name" with the name of your scenario.
+- _apply/scenario.yaml_: This file specifies all Kubernetes resources such as namespaces, deployments and services which are part of the scenario. This Yaml will be deployed when the scenario is started. [Example scenario.yaml](https://github.com/kubernetes-simulator/simulator/blob/master/simulation-scripts/scenario/network-hedgehog-defence/apply/scenario.yaml).
+- _challenge.txt_: This file contains metadata for the scenario that you want to make available to the user, such as scenario description, starting point, difficulty level and the descriptions of the tasks. This will get displayed on entering the scenario as the challenge text to the user. The _challenge.txt_ can be templated by using any of the environment variables available to the attack container, for example `$MASTER_IP_ADDRESSES`. Also, `##IP` can be prefixed to a Deployment, Pod or DaemonSet name to insert the pod IPs for their associated pods. For example, `##IPfrontend` would be substituted with pod IPs for all pods with names containing 'frontend'. `##NAME` will insert the full pod names and `##HIP` will insert the host IPs of the pods. Example [challenge.txt](https://github.com/kubernetes-simulator/simulator/blob/master/simulation-scripts/scenario/network-hedgehog-defence/challenge.txt).
+- _tasks.yaml_: This file contains the task and hint specs for all tasks in the scenario. It also determines where you start - e.g. in a pod on the cluster, on a node or even outside the cluster. [Example tasks.yaml](https://github.com/kubernetes-simulator/simulator/blob/master/simulation-scripts/scenario/network-hedgehog-defence/tasks.yaml). To see the required structure and options of the tasks.yaml, see the guide [here](https://github.com/kubernetes-simulator/simulator/blob/master/docs/tasks-yaml-format.md).
+
+Additionally the `simulator/simulation-scripts/scenario/` directory can include a number of supporting bash scripts which can we executed on the master and worker nodes. These files help with particular setups on the nodes such as installing tooling, creating files or setting up users. The scripts are executed on startup and must have the following names:
+
+- _worker-any.sh_: runs on a randomly chosen worker node
+- _worker-1.sh_: runs on worker node 1
+- _worker-2.sh_: runs on worker node 2
+- _workers-every.sh_: runs on worker nodes 1 & 2
+- _nodes-every.sh_: runs on master and worker nodes 1 and 2
+- _master.sh_: runs on master
+
+Any other names will cause an error. See an example `[master.sh](http://master.sh)` script [here](https://github.com/kubernetes-simulator/simulator/blob/master/simulation-scripts/scenario/etcd-inverted-wedge/master.sh).
```

Comment:
Is this intended to literally print `[master.sh](http://master.sh)`?

Reply:
good spot